models/README.md

```diff
@@ -7,3 +7,27 @@ servers. These diffs are generated against the standalone-full-ha.xml profile, w
 
 This data allows the Wildscribe pages to be generated offline, without a running server instance. It also means that at some
 stage it should be possible to generate diff's of the models.
+
+Add the XTS subsystem
+=====================
+Add the following to standalone-full-ha.xml:
+
+  <extension module="org.jboss.as.xts"/>
+
+  <subsystem xmlns="urn:jboss:domain:xts:2.0">
+      <host name="default-host"/>
+      <xts-environment url="http://${jboss.bind.address:127.0.0.1}:8080/ws-c11/ActivationService"/>
+      <default-context-propagation enabled="true"/>
+  </subsystem>
```

Comment:
Can this be a reference to the dist's docs/examples/configs/standalone-xts.xml instead of a static config chunk?